lib/graphql/schema/type_reducer.rb

```diff
@@ -1,10 +1,19 @@
 # frozen_string_literal: true
+
+require 'active_support/inflector'
+
 module GraphQL
   class Schema
-    module ReduceTypes
+    class TypeReducer
```

Comment:
Ah damn, you're right about backwards-compatibility 🤔 

Yeah I'll just add `camelize` to all functions not a big deal for now!